.gitignore

```diff
@@ -6,7 +6,9 @@
 /.gomodcache
 /html
 /Infra.toml
+/Test.toml
 /testsys.kubeconfig
+/testsys.kubeconfig.eksctl.lock
```

Comment:
I don't think we should add this here. How is this lock file generated? From my experience, it's only when I create a new cluster with `eksctl`. This lock file only shows up under pretty specific use-cases (i.e. using eksctl to create the testsys cluster in the root dir of this repo).

Reply:
Please remove `/testsys.kubeconfig.eksctl.lock` from `.gitignore`. It's not specific to Bottlerocket or any of its build/test frameworks.

For developers who don't want `eksctl.lock` files show up in show up any of their git trees, they can use their personal `.gitignore` in `$HOME/.gitignore`